frontend/src/v5/store/projects/projects.selectors.ts

```diff
@@ -19,6 +19,7 @@ import { createSelector } from 'reselect';
 import { selectCurrentTeamspace } from '../teamspaces/teamspaces.selectors';
 import { IProjectsState } from './projects.redux';
 import { IProject } from './projects.types';
+import { DEFAULT_STATUS_CONFIG } from '@controls/chip/chip.types';
```

Comment:
Status is not showing when opening the ticket
<img width="1137" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/cea43525-4592-484b-a7a6-1037f3a2483d">